mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.erb

```diff
@@ -4207,4 +4237,80 @@ resource "google_container_node_pool" "without_confidential_boot_disk" {
 }
 `, cluster, networkName, subnetworkName, np)
 }
+
+func testAccContainerNodePool_resourceManagerTags(projectID, clusterName, networkName, subnetworkName, randomSuffix string) string {
```

Comment:
The indentation should be two spaces in the resource configurations.

Reply:
done